tachyon/zk/plonk/vanishing/circuit_polynomial_builder.h

```diff
@@ -119,8 +121,16 @@ class CircuitPolynomialBuilder {
         if (permutation_provers_[j].grand_product_polys().size() > 0)
           UpdatePermutationCosets(j);
         // Do iff there are lookup constraints.
-        if (lookup_provers_[j].grand_product_polys().size() > 0)
-          lookup_evaluator.UpdateLookupCosets(*this, j);
+        size_t lookup_polys_num = 0;
+        if constexpr (LS::type == lookup::Type::kHalo2) {
```

Comment:
Let's fix this to switch statement.

Reply:
Since switch can't be `constexpr`. I think it's nicer.